app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,29 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{width: "100%"}
```

Comment:
Can we change this to:
```ruby
%table.social.white-bg.fullwidth
```
and drop the width declaration at the end?

Reply:
Of course :+1: